noir-projects/aztec-nr/aztec/src/macros/functions/mod.nr

```diff
@@ -1,14 +1,10 @@
-pub mod interfaces;
-pub mod initialization_utils;
```

Comment:
I found this file very hard to read so I decided to keep here only the public functions and moved the rest to other files.